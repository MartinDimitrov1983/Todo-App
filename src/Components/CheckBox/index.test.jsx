import React from 'react';
import { render } from '@testing-library/react';

import Checkbox from './index';

test('Snapshot Checkbox', () => {
  const { asFragment } = render(
    <Checkbox label="Test" checked={true} onChange={() => {}} />,
  );
  expect(asFragment()).toMatchSnapshot();
});
