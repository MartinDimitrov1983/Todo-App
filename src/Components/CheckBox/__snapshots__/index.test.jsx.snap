// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Checkbox 1`] = `
<DocumentFragment>
  <div
    class="checkBoxWrapper"
  >
    <label
      class="label"
    >
      <input
        checked=""
        class="input"
        type="checkbox"
      />
      <span>
        Test
      </span>
    </label>
  </div>
</DocumentFragment>
`;
